tests/cases/focus/keyboard_focus.slint

```diff
@@ -0,0 +1,100 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import { Button, SpinBox, LineEdit, HorizontalBox, VerticalBox, TabWidget } from "std-widgets.slint";
+
+App := Window {
+    preferred-width: 500px;
+    preferred-height: 400px;
+    title: "Keyboard Focus Test";
+    icon: @image-url("../../logo/slint-logo-small-light-128x128.png");
+
+    property<[{name: string, account: string, score: float}]> model: [
+        {
+            name: "Olivier",
+            account: "ogoffart",
+            score: 456,
+        },
+        {
+            name: "Simon",
+            account: "tronical",
+            score: 789,
+        }
+    ];
+
+    VerticalBox {
+        HorizontalBox {
+            VerticalLayout {
+                alignment: start;
+                SpinBox {
+                    value: 42;
+                }
+            }
+        }
+
+        TabWidget {
+            Tab {
+                title: "Dynamic";
+
+                VerticalBox {
+                    SpinBox {
+                        value: 58;
+                    }
+                    for person[i] in model: HorizontalBox {
+                        property<int> index;
+
+                        SpinBox {
+                            value: person.score;
+                        }
+                        Button {
+                            text: "Test " + index;
+                        }
+                        SpinBox {
+                            value: i;
+                        }
+                    }
+                    SpinBox {
+                        value: 59;
+                    }
+                }
+            }
+            Tab {
+                title: "Static";
+                VerticalBox {
+                    HorizontalBox {
+                        alignment: start;
+                        VerticalBox {
+                            alignment: start;
+                            SpinBox {
+                                value: 10;
+                            }
+                            SpinBox {
+                                value: 11;
+                            }
+                            SpinBox {
+                                value: 12;
+                            }
+                        }
+
+
+                        VerticalBox {
+                        alignment: start;
+                            SpinBox {
+                                value: 20;
+                            }
+                            SpinBox {
+                                value: 21;
+                            }
+                            SpinBox {
+                                value: 22;
+                            }
+                        }
+                    }
+                    LineEdit {
+                        placeholder-text: "Enter some text";
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
You can use some javascript here and simulate the tab and see who has focus.

Reply:
Yes, the plan is to turn this into a proper test:-)

Next step after making forward/backward tab working with interpreter/rust/cpp is to figure out where the focus vanishes to. Some widgets seem to accept focus but do not show that in any way.